tests/file_storage/tests.py

```diff
@@ -944,6 +944,18 @@ def test_filefield_default(self):
         self.assertEqual(obj.default.read(), b"default content")
         obj.default.close()
 
+    def test_filefield_db_default(self):
+        temp_storage.save("tests/db_default.txt", ContentFile("default content"))
+        obj = Storage.objects.create()
+        self.assertEqual(obj.db_default.name, "tests/db_default.txt")
+        self.assertEqual(obj.db_default.read(), b"default content")
+        obj.db_default.close()
+
+        obj.delete()
+        obj = Storage()
```

Comment:
Do you think it'll be worth making this either two tests or a subTest-formatted test?

Reply:
I originally thought these were 2 independent test conditions/assertions but now with the added comment, I think I understand how the deletion is part of the tested flow.